docs/guides/noise-learning.ipynb

```diff
@@ -0,0 +1,194 @@
+      " 'ZX', 'ZY', 'ZZ'], rates=[0.00037, 0.00046, 0.00035, 0.00127, 0.00067, 0.0, 0.0, 0.00067, 0.00127, 0.0, 0.0, 0.00071, 0.0, 0.00035, 0.00046])\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f'Noise learner result contains {len(result.data)} entries and has the following data:\\n {result}\\n')\n",
+    "print(f'The layer error model for the first layer has data:\\n {result.data[0]}\\n')\n",
+    "print(f'And the Pauli Lindblad model has has the following data: \\n{result.data[0].error}\\n')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a27a38f1-393d-49f9-92c3-a8634bfb1a60",
+   "metadata": {},
+   "source": [
+    "The `LayerError.error` attribute of the noise learning result contains the generators and error rates of the fitted Pauli Lindblad model, which has the form:\n",
+    "\n",
+    "$$ \\Lambda(\\rho) = \\exp{\\sum_j r_j \\left(P_j \\rho P_j^\\dagger - \\rho\\right)} $$\n",
+    "\n",
+    "where the $r_j$ are the `LayerError.rates` and $P_j$ are the Pauli operators specified in `LayerError.generators`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ebb4b7d-ef45-4996-bf5a-125d0831ebca",
+   "metadata": {},
+   "source": [
+    "## Noise learning options\n",
+    "\n",
+    "There are a few options available to input into a `NoiseLearner` object, can be set with when instantiated. These are encapsulated by the `qiskit_ibm_runtime.options.NoiseLearnerOptions` class and include:\n",
+    "\n",
+    "- `max_layers_to_learn`: The maximum number of unique entangling layers to learn (the default value is `4` and can be set to `None` to indicate no limit on the number of layers)\n",
+    "- `num_randomizations`: Total number of random circuits to use per learning circuit configuration\n",
+    "- `shots_per_randomization`: Total number of shots per learning circuit\n",
+    "- `layer_pair_depths`: The circuit depths to us in the learning experiments (measured in number of pairs)\n",
+    "- `twirling_strategy`: The twirling strategy used in the layers of two-qubit twirled gates. The allowed values are:\n",
+    "    - `\"active\"`: only instruction qubits are twirled in each individual twirled layer\n",
+    "    - `\"active-circuit\"`: in each twirled layer, the union of all qubits with an instruction are twirled\n",
+    "    - `\"active-accum\"`: in each twirled layer, the union of all qubits with an instruction in the circuit *up to the current layer* are twirled\n",
+    "    - `\"all\"`: in each twirled layer, *all* qubits in the circuit are twirled\n",
+    "\n",
+    "Below is a simple example showing how to use the `NoiseLearnerOptions` in a `NoiseLeaner` experiment:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8f35d68a-b459-4f1c-ae44-a01bbe700ed0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Build a larger GHZ circuit\n",
+    "qubits = list(range(10))\n",
```

Comment:
I think that you may just want to have this:
```
circuit = QuantumCircuit(10)
circuit.h(0)
circuit.cx(range(0, 10, 2), range(1, 10, 2))
```

Reply:
Good point thanks! Added this to the upcoming commit.